services/libfuzzer/README.md

```diff
@@ -26,12 +50,13 @@ It is recommended to reserve at least 4GB of memory for containers running cover
 ### Example: LibFuzzer Run
 
 ```bash
-fuzzfetch --fuzzing --coverage -a --gtest -n firefox
+fuzzfetch --fuzzing --coverage --gtest -n firefox
```

Comment:
Remove `--coverage`

Reply:
Also, there doesn't appear to be an fuzzing-opt so maybe use `-a --fuzzing --gtest`